### ISSUE TITLE
Current location integration with Android Location/Google Maps

### ISSUE THREAD

**Issue by ppeters0502** [open-source-ideas/national-weather-service-android-app#2]
Currently working on integrating with Google maps, so that longitude and latitude that are supplied to API call come from current location rather than manual entry.

**Comment by saiteja09** [comment]
Do we need Google Maps integration to get the latitude and longitude. I think, we can use Android's native API.
https://developer.android.com/reference/android/location/Location

**Comment by ppeters0502** [comment]
Originally I had thought about using Android Location, but I wasn't sure if in using that, you'd have the option to select a different area (such as finding a city on a map) and use the latitude and longitude of the location you selected. That way, someone using the app could either use their phone's location, or select a different location.
To be honest, I wasn't sure if that was supported on Android Location, if it is then we can use the native Android API.

**Comment by saiteja09** [comment]
I was thinking, instead of using a map for the user to select the location, wouldn't it be more natural to just type in the location and autofill it, just like all the weather apps. We can use the Places API to autofill the city name and use that to get latitude and longitude.

https://developers.google.com/places/web-service/autocomplete

Let me know what you think about this.

**Comment by ppeters0502** [comment]
Yeah, I like that! Easier for the user, and easier to put together!
